### ISSUE TITLE
CR instead of LF line endings prevents from running on Mac

### ISSUE THREAD

**Issue by hulbert** [partageit/markdown-to-slides#12]
Running the following on Mac OS X:

```sh
mkdir testing
npm init -f
npm install --save markdown-to-slides
$(npm bin)/markdown-to-slides 
```

Encounters the following error:

```
env: node\r: No such file or directory
```

Modifying the `index.js` file to save w/ LF fixes the issue.

**Comment by higgser** [comment]
I run into the same problem on my Mac OS X when I followed the instructions. When I make a checkout and install with git and npm, I don't have the problem.

```
git clone https://github.com/partageit/markdown-to-slides.git
cd markdown-to-slides
npm install -g
cd ..
rm -rf markdown-to-slides
```

**Comment by partageit** [comment]
@hulbert  and @higgser, thank you for your analysis :) 

I made changes to `index.js` in the latest version to keep LF in `index.js`.
I guess/hope it works on mac os X. Could you confirm that the problem is fixed with `markdown-to-slides 1.0.5`?

**Comment by hulbert** [comment]
@partageit Yep, appears fixed w/ 1.0.5!

**Comment by partageit** [comment]
Thank you for your feedback :)
